docs/docs/extension.md

```diff
@@ -366,8 +365,7 @@ Check the [Custom Upstream auth guide](./guides/authentication/custom-auth-upstr
 x-kusk:
 ...
   auth:
-    scheme: cloudentity
-    auth-upstream:
+    auth-cloudentity:
```

Comment:
Why not just `cloudentity` since it's already under the `auth` key?

Reply:
I think I missed this